addons/crm/models/res_partner.py

```diff
@@ -44,6 +44,15 @@ def _fetch_children_partners_for_hierarchy(self):
     def _get_contact_opportunities_domain(self):
         return [('partner_id', 'in', self._fetch_children_partners_for_hierarchy().ids)]
 
+    @api.depends('name')
+    @api.depends_context('crm.partner_display_name_hide_company')
+    def _compute_display_name(self):
```

Comment:
We could add it in base/res_partner.py, in the base compute_display_name so that it is available for all modules if necessary (also to have a "complete" definition in a single method).